UnicodeJsps/src/main/java/org/unicode/jsp/CachedProps.java

```diff
@@ -44,8 +44,7 @@ public class CachedProps {
     final BiMultimap<String, String> nameToAliases = new BiMultimap<String, String>(null, null);
     final Map<String, BiMultimap<String, String>> nameToValueToAliases = new LinkedHashMap();
 
-    static String jspVersion = IS_BETA ? Settings.lastVersion : Settings.latestVersion;
-    static VersionInfo jspVersionInfo = VersionInfo.getInstance(jspVersion);
+    static VersionInfo jspVersionInfo = UCharacter.getUnicodeVersion();
```

Comment:
FYI
I realize that I had suggested the code that you are fixing here now.
I hadn't realized that we were currently serving ICU property values...

Reply:
Right, I did not realize that either.